libevm/VMSIMD.cpp

```diff
@@ -538,107 +537,106 @@ void VM::xget(uint8_t _b, uint8_t _c)
 	}
 }
 
-void VM::xput(uint8_t _b, uint8_t _c)
+void VM::xput(uint8_t _src_type, uint8_t _dst_type)
```

Comment:
A grep on `_[a-z]+_[a-z]+` also smoked out some `_o_out` and `_stack_item` hiding in plain sight.